test/logging/request-obfuscator_test.js

```diff
@@ -0,0 +1,94 @@
+
+/**
+ * Module dependencies.
+ */
+
+import { obfuscate } from '../../src/logging/request-obfuscator';
+
+/**
+ * Test `RequestObfuscator`.
+ */
+
+describe('RequestObfuscator', () => {
+  describe('obfuscate', () => {
+    it('should not obfuscate `request.body.params` when `method` is not listed for obfuscation', () => {
+      const request = { body: '{"id":"1485369469422","method":"foo","params":["foobar"]}' };
+
+      obfuscate(request);
+
+      request.should.eql({ body: '{"id":"1485369469422","method":"foo","params":["foobar"]}' });
+    });
+
+    it('should obfuscate the authorization from `request.headers`', () => {
+      const request = { headers: { authorization: 'Basic ==foobar' } };
+
+      obfuscate(request);
+
+      request.should.eql({ headers: { authorization: 'Basic ******' } });
+    });
+
+    it('should obfuscate the private key from `response.body` when `method` is equal to `dumpprivkey`', () => {
```

Comment:
Move this test to the end of the test suite? That way we test request.body and then the response.body.